src/components/qualitySelector/qualitySelector.js

```diff
@@ -0,0 +1,116 @@
+import * as djs from 'dashjs';
+import 'videojs-per-source-behaviors';
+import 'videojs-contrib-quality-levels';
+import 'videojs-http-source-selector';
+
+import './quality-selector.scss';
+
+const qualitySelector = {
+  init: (player) => {
```

Comment:
Can you please add general documentation for each function - what it does, when/by what is it used and also some internal documentation to explain each part? thanks!

Reply:
done